.gitignore

```diff
@@ -26,6 +26,4 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
-# Ignore test-related files
-/coverage
-/coverage/
+coverage
```

Comment:
Awesome job! We finally got this working

Reply:
Thank you!